File: 1. Basic Topics/1. Arrays/py/1431. Kids With the Greatest Number of Candies.py
class Solution:
    def kidsWithCandies(self, candies: List[int], extraCandies: int) -> List[bool]:
        highest = max(candies)
        returnArr = []
        for i in candies:
            if (i+extraCandies) >= highest:
                returnArr.append(True)
            else:
                returnArr.append(False)

        return returnArr

File: 1. Basic Topics/1. Arrays/cpp/1431. Kids With the Greatest Number of Candies.cpp
class Solution {
public:
    vector<bool> kidsWithCandies(vector<int>& candies, int extraCandies) {
        int length = candies.size();
        vector<bool> returnVector;
        int max_score = *max_element(candies.begin(), candies.end());
        for(int i = 0; i<length; i++){
            if(candies[i]+extraCandies>=max_score){
                returnVector.push_back(true);
            }else{
                returnVector.push_back(false);
            }
        }
        return returnVector;
    }
};